ocaml/xapi/xapi_xenops.ml

```diff
@@ -189,15 +189,28 @@ let firmware_of_vm vm =
         bad]))
   | exception Not_found -> default_firmware
 
+let varstore_rm_with_sandbox ~__context ~vm_uuid f =
+  let dbg = Context.string_of_task __context in
+  let domid = 0 in
+  let chroot, socket_path = Xenops_sandbox.Varstore_guard.start dbg ~domid ~vm_uuid ~paths:[] in
+  Xapi_stdext_pervasives.Pervasiveext.finally (fun () -> f chroot socket_path)
+    (fun () ->
+       let (_: string list) = Xenops_sandbox.Varstore_guard.stop dbg ~domid ~paths:[] in ())
```

Comment:
How about `ignore @@ Xenops_sandbox.Varstore_guard.stop dbg ~domid ~paths:[]`

Reply:
I agree that this looked ugly, I've actually fixed this in the xenopsd PR by making `stop` return unit, updated this PR too now.